hubspot/connection/exc.py

```diff
@@ -32,9 +32,11 @@ class HubspotClientError(HubspotException):
     :param unicode request_id:
```

Comment:
The docstring should be updated with `error_message`, I don't understand why it wasn't there for `msg` in the first place :confused: